### ISSUE TITLE
training scripts

### ISSUE THREAD

**Issue by lucasjinreal** [declare-lab/nora-1.5#1]
hi, will consider open training scripts?

**Comment by soujanyaporia** [comment]
Yes, we are releasing it in a week.

**Comment by lucasjinreal** [comment]
awesome.

**Comment by oo1232** [comment]
> Yes, we are releasing it in a week.

Thank you very much for the excellent work. Could you provide an update timeline for the release of the training code?

**Comment by soujanyaporia** [comment]
There is a training branch you can take a look but we have yet to test it. Sorry for the delay. We are working hard to release it asap.

**Comment by oo1232** [comment]
> There is a training branch you can take a look but we have yet to test it. Sorry for the delay. We are working hard to release it asap.

Thank you for your response and for your diligent efforts. Looking forward to the release of your official training code！

**Comment by soujanyaporia** [comment]
The training code has been released. Sorry for the wait.

**Comment by oo1232** [comment]
> The training code has been released. Sorry for the wait.

Thank you very much for the update. However, it appears that your codebase currently only includes the implementation for Phase I: Action-Expert Training. Could you please let me know where the code for Phase II: Reward-guided Post-Training is located?

**Comment by soujanyaporia** [comment]
We are trying to release these codes soon.

**Comment by hungchiayu1** [comment]
Training code has been released
